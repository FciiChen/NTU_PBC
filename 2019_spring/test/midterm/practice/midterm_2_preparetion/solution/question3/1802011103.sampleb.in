midterm2_q3_ds2.csv