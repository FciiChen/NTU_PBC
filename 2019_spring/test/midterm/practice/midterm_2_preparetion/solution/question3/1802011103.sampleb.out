A1101,alpha,0112,****----****
A1102,beta
A1103,gamma,34.240
B1101,eta
